packages/probing-frontend/src/pages/MonitoringDetailPage/components/MonitoringEserviceProbing.tsx

```diff
@@ -0,0 +1,118 @@
+import type { ProbingEservice } from '@/api/monitoring/monitoring.models'
+import { Alert, Box, Chip, Grid, Typography } from '@mui/material'
+import { ButtonNaked } from '@pagopa/mui-italia'
+import { useTranslation } from 'react-i18next'
+import RefreshIcon from '@mui/icons-material/Refresh'
+import { formatDateString } from '@/utils/date.utils'
+import type { TranslationKeys } from './MonitoringInformationContainer'
+import { MonitoringInformationContainer } from './MonitoringInformationContainer'
+
+export const MonitoringEserviceProbing = ({
+  eservicesProbingDetail,
+  refetch,
+}: {
+  eservicesProbingDetail: ProbingEservice | undefined
+  refetch: () => Promise<void>
+}) => {
+  const { t } = useTranslation('common', {
+    keyPrefix: 'detailsPage',
+  })
+
+  const getChipColor = (value: string) => {
```

Comment:
The value could be of type `ProbingStatusType` exported from `monitoring.models.ts`